packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts

```diff
@@ -102,6 +102,15 @@ export const erc20FeeProxyArtifact = new ContractArtifact<ERC20FeeProxy>(
         },
       },
     },
+    '0.2.0-next': {
```

Comment:
what is this?

Reply:
I did not want to change the version just for that but I also wanted to document the new address. Do you think it's a bad idea?

Reply:
hum maybe just add a comment...

Reply:
You mean instead of the entry or to explain the entry?

Reply:
both :D